cypress/e2e/integration/MainPage.cy.ts

```diff
@@ -0,0 +1,95 @@
+import { DateProps } from "interfaces/date_props.interface";
+
+describe("Renders Main page of app", () => {
+  const lengthOfItems = 5;
+  it("Renders correctly", () => {
+    cy.visit("/");
+  });
+
+  it("Testing date", () => {
+    const today = new Date(Date.now());
+    const convertedDate: DateProps = {
+      day: today.getDate(),
+      month: today.toLocaleString("default", { month: "long" }),
+      year: today.getFullYear(),
+      currentDay: today.toLocaleString("default", { weekday: "long" }),
+    };
+    cy.findByTestId("header-day").should("have.text", convertedDate.day);
+    cy.findByTestId("header-month").should("have.text", convertedDate.month);
+    cy.findByTestId("header-year").should("have.text", convertedDate.year);
+    cy.findByTestId("header-day-name").should(
+      "have.text",
+      convertedDate.currentDay
+    );
+  });
+
+  it("Testing CRUD on 1 Task", () => {
+    cy.findByTestId("task-input-field").type("Item 1");
+    cy.findByTestId("create-task").click();
+    cy.findByTestId("created-tasks-list").children().should("have.length", 1);
+    cy.findByTestId("task-title").should("have.text", "Item 1");
+    cy.findByTestId("pause-task").click();
+    cy.findByTestId("finish-task").click();
+    cy.findByTestId("remove-task").click();
+  });
+
+  it(`Testing Local storage and CRUD on ${lengthOfItems} tasks`, () => {
```

Comment:
I don't see a local storage test. If the action influences the local storage, maybe add a test to check it too.

Reply:
cy.reload() has been reloading the page again and then it's fetching data from local storage

Reply:
Pls pull the local storge data and verify that all render to the dom.

Reply:
II'l create another 'it' for this one.